participant/views.py

```diff
@@ -127,3 +127,13 @@ def action(self, request, *args, **kwargs):
 
         resp['action_id'] = str(participant_action.id)
         return Response(data=resp)
+
+
+class ParticipantScoreViewSet(viewsets.ModelViewSet):
+    queryset = Participant.objects.all()
+    http_method_names = ('get', 'head', 'options')
+
+    def get_serializer_class(self):
+        if self.action == 'list':
+            return ParticipantScoreSerializer
+        return ParticipantScoreSerializer
```

Comment:
If you want to use the same serialiser for both list and details you can remove the getter function and just define `serializer` as normal after `queryset`
```
serializer = ParticipantScoreSerializer
```